src/main/java/edu/kis/powp/jobs2d/command/visitor/IVisitor.java

```diff
@@ -0,0 +1,11 @@
+package edu.kis.powp.jobs2d.command.visitor;
+
+import edu.kis.powp.jobs2d.command.ICompoundCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+public interface IVisitor {
```

Comment:
The name should suggest that Visitor concerns DriverCommand hierarchy (e.g., DriverCommandsVisitor)